platforms/web/public/locales/es/account.json

```diff
@@ -204,7 +204,8 @@
     "password_reset_success": "Contraseña restablecida",
     "password_reset_text": "",
     "password_strength": "Aunque tu contraseña cumple con los criterios de una contraseña segura, incluye elementos que no son recomendables, como frases prohibidas o palabras repetidas. Por favor, modifícala para garantizar la seguridad de tu cuenta.",
-    "passwords_do_not_match": "Las contraseñas no coinciden",
+    "passwords_do_not_match": "La contraseña antigua no coincide",
```

Comment:
I think this is a mistake, this translation should not be updated.

Reply:
But it's the value for "old_password_does_not_match" actually.

Reply:
Sharp, has been fixed!